doctor/cores/text_processor.py

```
@@ -0,0 +1,462 @@
+        # شمارش کلمات پزشکی
+        for word in text.split():
+            if word.strip('.,!?():"') in self.medical_terms:
+                medical_word_count += 1
+        
+        # اضافه کردن امتیاز موجودیت‌ها
+        entity_score = len(entities) * 0.1
+        
+        relevance = (medical_word_count / total_words) + entity_score
+        return min(relevance, 1.0)
+    
+    def _split_into_sentences(self, text: str) -> List[str]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)